cdap-ui/app/cdap/components/ConfigurationGroup/WidgetWrapper/index.tsx

```diff
@@ -108,6 +109,7 @@ const WidgetWrapperView: React.FC<IWidgetWrapperProps> = ({
   classes,
   errors,
   size = 'large',
+  hideLabel = false,
```

Comment:
I don't think it breaks anything, but might be good to add default value for hideDescription for readability